docs/COMMANDS.md

```diff
@@ -0,0 +1,141 @@
+# Commands
+GlueStick comes with several commands to help you get started. You can full list of commands available on gluestick below:
+
+### `gluestick new`
+
+Takes care of generating new project
+
+```bash
+gluestick new <YOUR_APPLICATION_NAME>
+```
+
+Available options:
+
+* `-d, --dev <path>` - Relative path to development version of gluestick
+* `-y, --yarn` - Use yarn to perform installations
+
+### `gluestick generate`
+
+Generates a new entity from given template
+
+```bash
+gluestick generate <ENTITY_TYPE> <ENTITY_NAME>
+```
+
+Available options:
+
+* `-E --entry-point <entryPoint>` - Entry point for generated files
+* `-F, --functional` - Generate stateless functional component
+* `-O, --gen-options <value>` - Options to pass to the generator
+
+### `gluestick start`
+
+Starts a gluestick project
+
+```bash
+gluestick start
+```
+
+Available options:
+
+* `-T, --run-tests` - Run test hook
+* `-L, --log-level <level>` - Set the logging level
+  * Valid options: `fatal`, `error`, `warn`, `info`, `debug`, `trace`, `silent`
+* `-E, --log-pretty [true|false]` - Set pretty printing for logging
+* `-D, --debug-server` - Debug server side rendering with built-in node inspector
+* `-p, --debug-port <number>` - Port on which to run node inspector
+* `-C --coverage` - Create test coverage report
+* `-P, --skip-build` - Skip build when running in production mode
+
+### `gluestick build`
+
+Create an production asset build
+
+```bash
+gluestick build
+```
+
+### `gluestick bin`
+
+Access dependencies bin directory
+
+```bash
+gluestick bin <DEPENDENCY_NAME> -- <DEPENDENCY_ARGS>
+```
+
+### `gluestick dockerize`
+
+Create docker image (requires `docker` installed on your machine)
+
+```bash
+gluestick dockerize
+```
+
+### `gluestick start-client`
```

Comment:
The 3 `start` commands should be together (probably next to `start` on the top) since they are very related.